src/components/job-details.tsx

```diff
@@ -98,8 +98,8 @@ export function JobDetails(props: JobDetailsProps): JSX.Element | null {
       </div>
       <div className={'jp-notebook-job-details-grid'}>
         <div className={rowClass}>
-          <div className={keyClass}>{trans.__('Start date')}</div>
```

Comment:
Shouldn't both start _and_ create be displayed in the details?

Reply:
Actually, this looks like the inline job details, which is disabled in the new job row — this whole file can probably be deleted if so.